src/quote.ts

```diff
@@ -0,0 +1,15 @@
+import { _toMarkdown } from './_toMarkdown.js'
+import { Markdown } from './markdown.js'
+
+/**
+ * Creates a Markdown block quote.
+ * @example
+ * md.quote('foo_bar') // => Markdown with value '>foo\\_bar'
+ */
+export const quote = (text: unknown): Markdown => {
+  if (!text) {
+    return _toMarkdown()
+  }
+
+  return _toMarkdown(`>${_toMarkdown(text).replaceAll('\n', '\n>')}`, true)
```

Comment:
Suggest adding `\n` here as well:
```suggestion
  return _toMarkdown(`>${_toMarkdown(text).replaceAll('\n', '\n>')}\n`, true)
```